simulation/dynamics.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022-, Semiotic AI, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+from multipledispatch import dispatch
+
+from simulation.entity.action import *
+from simulation.entity.state import *
+
+
+@dispatch(PriceState, PriceAction)
+def dynamics(s: PriceState, a: PriceAction) -> None:  # type: ignore
+    """Update the state given the action.
+
+    In this case, the new state is just the new action.
+
+    Arguments:
+        s (PriceState): The previous state
+        a (PriceAction): The current action
+    """
+    s.value = a.value
+
+
+@dispatch(PriceState, PriceMultiplierAction)
+def dynamics(s: PriceState, a: PriceMultiplierAction) -> None:  # type: ignore
```

Comment:
I really think that few words explanation what "dynamics" class does would be useful

Reply:
a more abstract explanation, I mean

Reply:
I understand it is mapping state->action

Reply:
So do you want me to document that dynamics map state to action? What do you mean by "more abstract explanation?" Happy to do it. Just don't know what you mean.